finality-aleph/src/sync/forest/mod.rs

```diff
@@ -293,18 +280,12 @@ impl<I: PeerId, J: Justification> Forest<I, J> {
 
     /// Updates the `highest_justified` if the given id is higher.
     fn try_update_highest_justified(&mut self, id: BlockIdFor<J>) -> bool {
-        match &self.highest_justified {
-            Some(current_id) => match id.number() > current_id.number() {
-                true => {
-                    self.highest_justified = Some(id);
-                    true
-                }
-                false => false,
-            },
-            None => {
-                self.highest_justified = Some(id);
+        match id.number() > self.highest_justified.number() {
```

Comment:
nit: if would be more readable here imho

Reply:
hmm, or you meant
```
    if id.number() > self.highest_justified.number()  {
        self.highest_justified = id;
        return true;
    }
    false
```
?

Reply:
imho there is less noise in the second variant but this is just an opinionated ~attack~ nit so feel free to resolve :P

Reply:
we <3 match, resolve it is